src/index.html

```diff
@@ -12,7 +12,7 @@
   <div class="Name">
     <input type="text" placeholder="Enter your Name" required>
     <input type="password" placeholder="Enter password" required>
-		<button type="button">Submit</button>
+		<button type="button">Submit Button</button>
```

Comment:
why you changed submit button name?

Reply:
As per client requirement, i have changed name